tools/db_stress.cc

```diff
@@ -2352,6 +2360,17 @@ int main(int argc, char** argv) {
   SetUsageMessage(std::string("\nUSAGE:\n") + std::string(argv[0]) +
                   " [OPTIONS]...");
   ParseCommandLineFlags(&argc, &argv, true);
+  rocksdb::SyncPoint::GetInstance()->SetCallBack(
```

Comment:
I'm still kind of confused why we run stress tests or unit tests for direct I/O on platforms that don't support it. And why does db_stress mask O_DIRECT on all platforms while db_test_util masks it on a subset of platforms?